Setup/UpgradeData.php

```diff
@@ -568,6 +569,8 @@ public function upgrade(ModuleDataSetupInterface $setup, ModuleContextInterface
 
         $this->setProductIDIN($setup);
 
+        $this->addCustomerLastPayByBankIssuer($setup);
```

Comment:
It is better to be moved in        if (version_compare($context->getVersion(), '1.45.2', '<')) {
            $this->addCustomerLastPayByBankIssuer($setup);
        }
and change also 1.45.2 setup_version from module.xml